textproc/less/files/patch-configure.diff

```diff
@@ -1,56 +0,0 @@
---- configure.orig	2019-06-11 13:12:02.000000000 -0500
```

Comment:
Was this applied upstream? Or why is it no longer needed?

Reply:
The patch added headers for the "checking for working terminal libraries..." checks. The new configure script has checks that look like this:
```
    cat confdefs.h - <<_ACEOF >conftest.$ac_ext
/* end confdefs.h.  */
$include_termcap_h
int
main ()
```
which puts a termcap.h include into that check. If you look in the GitHub actions build output you'll see it successfully found libncursesw to link against.

Reply:
Thank you for the explanation! That looks very good.